Documentation/gettingstarted/k3s.rst

```diff
@@ -16,8 +16,8 @@ workloads in unattended, resource-constrained, remote locations or inside IoT
 appliances.
 
 This guide assumes installation on amd64 architecture. Cilium is presently
-supported on amd64 architecture with `ARM support planned <https://github.com/cilium/cilium/issues/9898>`_
-for a future release.
+supported on amd64 architecture with ARM support planned for a future release
+(See :gh-issue:`9898`).
```

Comment:
Not related to this PR, but we should be good to remove / update this given that we will ship arm64 containers in v1.10.x.

Reply:
Right, I haven't checked the status of the different GH issues referenced.
I can update this paragraph, do you know if there are specific steps for arm64, or if the whole comment should go?

Reply:
Maybe given the k3s context here it'd be worth mentioning that amd64 and arm64 are supported. That should be enough to imply that arm32 is not supported in case anyone is thinking about that (although I *think* upstream might be thinking about removing such support).

Reply:
Ok thanks, I updated it in a new commit, for clarity.